### PR TITLE
Chapter 11: Blade Template

### PR DESCRIPTION
**Pull request by dangvm-0369**

### CODE REVIEW

**Comment by dangvm-0369** [comment]
ready

File: resources/views/users/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    </h3>
+                    <table class="table mt-4">
+                        <thead>
+                        <tr>
+                            <th class="text-gray-900 dark:text-gray-100" scope="col"> User Name </th>
+                            <th class="text-gray-900 dark:text-gray-100" scope="col"> First Name </th>
+                            <th class="text-gray-900 dark:text-gray-100" scope="col"> Last Name </th>
+                            <th class="text-gray-900 dark:text-gray-100" scope="col"> Email </th>
+                            <th class="text-gray-900 dark:text-gray-100" scope="col"> Roles </th>
+                            <th class="text-gray-900 dark:text-gray-100" scope="col"> Action </th>
```
Comment (ghost) [inline resources/views/users/show.blade.php]: localization

**Comment by dangvm-0369** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**